Makefile

```diff
@@ -348,6 +356,11 @@ UK_HAVE_DOT_CONFIG := y
 endif
 endif
 
+ifneq ($(CONFIG_HAVE_DRUNTIME), y)
```

Comment:
If we were to have a "user space" D program that uses the runtime, and an internal library written in D, we'd end up with the internal library using the runtime, which wouldn't work.